docs/notebooks/quickstart.ipynb

```diff
@@ -432,6 +432,85 @@
     "get_assessment_names()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "2d1e3de8-6d4a-4aed-82f3-c04fa701b949",
+   "metadata": {},
+   "source": [
+    "#### Assessment Documentation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db8ed40f-578a-4586-b0a5-28c680db7686",
+   "metadata": {},
+   "source": [
+    "Documentation for each assessment is a bit difficult to understand. Assessments wrap modules, which themselves have documentation. Normally, you don't have to worry about the module itself, except if you are creating your own assessments or want to use the modules directly. The main exception here is if you want to parameterize how you run the assessment\n",
```

Comment:
Calling it explicitly that documentation for each assessment is difficult to get. I think removing the `init_module` function may help here. But it's still a bit confusing.